protos/storage.proto

```diff
@@ -17,8 +17,13 @@ service StorageService {
   rpc PromQuery(prometheus.PrometheusQueryRequest) returns (prometheus.PrometheusQueryResponse) {}
 }
 
+message RequestHeader {
```

Comment:
RequestContext

IMO A request doesn't have header and body part

Reply:
done